platformio.ini

```diff
@@ -12,7 +12,8 @@
 framework = arduino
 platform = https://github.com/platformio/platform-espressif32.git
 build_flags =
-	-std=gnu++17
+	-std=gnu++2a
```

Comment:
Added this so it builds on Windows (needs to be checked is still builds on other systems)